docs/rpc.md

```diff
@@ -238,6 +238,19 @@ block. It can be removed in future versions, but at the moment you can use it
 to see how much GAS is burned with a particular block (because system fees are
 burned).
 
+#### `getrawnotarypool` and `getrawnotarytransaction` calls
+
+The `getrawnotarypool` method provide the ability to retrieve a list of hashes
+for the current notary transactions. The returned structure consists of `Hashes`.
```

Comment:
`... to retrieve the content of the RPC node's notary pool (a map from main transaction hashes to the corresponding fallback transaction hashes for currently processing P2PNotaryRequest payloads).`

Reply:
Fixed.